src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -10,72 +9,96 @@ import {
   getActiveExpandedPages,
 } from '../helpers';
 
+import { focusElement } from '../utilities/ui';
+
 import PropTypes from 'prop-types';
 import { REVIEW_APP_DEFAULT_MESSAGE } from '../constants';
 
-export default class FormNav extends React.Component {
-  // The formConfig transforming is a little heavy, so skip it if we can
-  shouldComponentUpdate(newProps) {
-    return !shallowEqual(this.props, newProps);
-  }
-  render() {
-    const { formConfig, currentPath, formData } = this.props;
+export default function FormNav(props) {
+  const { formConfig, currentPath, formData } = props;
+
+  const [index, setIndex] = useState(0);
```

Comment:
Names are tricky. This `index` is designed to be one integer behind the `current` const, which represents the current chapter. When `current` changes, this means that the `index` will change too: +1 for going to a new chapter, and -1 for going to a previous chapter.